### ISSUE TITLE
Scale fluid link sizes in Flow

### ISSUE THREAD

**Issue by dcbroad3** [factoriolab/factorio-lab#447]
Fluids appear too large in the sankey flow diagram when viewing by items, the default. Generally, 10 units of a fluid are comparable to 1 unit of an item, so dividing fluid counts by 10 (or making this a setting) would make the diagram appear more realistic.

**Comment by lilred** [comment]
>Generally, 10 units of a fluid are comparable to 1 unit of an item

- A cargo wagon carries 40 stacks (2000-4000 of most items). Fluid wagons carry 25000 items. So from the point of view of rail, one item is comparable with 6.25-12.5 fluid units.
- A barrel holds 50 units of liquid, and you need to carry it both ways. So you could argue that from the point of view of bots and belts an item is comparable with 25 fluid units. However barrelling and unbarrelling isn't free, it requires infrastructure, so let's say that 25 fluit units is a generous upper bound, and the range of "correct" values probably live below 20.
- Sulphur is made from 30 units of liquid. Plastic is made from 10 units of liquid and 0.5 solid items. Most recipes for intermediate items in the base game make one item from 2-10 solid items. In particular, intermediate items contemporary with plastic and sulphur usually require 2-4 solids to make. This suggests that the "correct" number range is in the vicinity of 5-15.

In conclusion, all reasonable hints cluster around 10, and 10 is a nice round value that's easy to visually interpret, so I endorse it. Would you welcome a PR doing this?

**Comment by dcbroad3** [comment]
I think it's also telling that pumpjack production is 10 * yield percentage. 
I would welcome a PR, but I'm also going to take a quick look myself because this might be fairly trivial.

**Comment by lilred** [comment]
Awesome!
